backend/endpoints/messages.py

```diff
@@ -129,8 +178,9 @@ async def update_message(
             status_code=status.HTTP_404_NOT_FOUND, detail="Message not found"
         )
 
-    request.edited = True
     payload = request.dict(exclude_unset=True)
+    message["richUiData"] = payload["richUiData"]
+    payload["edited"] = True
 
     if message["sender_id"] != payload["sender_id"]:
```

Comment:
@pauline-banye sorry please, was doing last minute checks before clicking approved and saw this line. Is this check supposed to come after line 182 or before?

Reply:
Hi @mykie88,

Lines 182 and 183 simply outline the payload data. 183 changes the "edited" field to True and 182 swaps the original data with the payload data. This was added so that the full updated object was returned in the response, as Aniebiet specified, without having to read zccore again.

Reply:
@pauline-banye what I am saying is if message["sender_id"] != payload["sender_id"] on line 185 should come before swapping the "richUiData" on line 182.

Reply:
@mykie88 it's not really a major concern as it won't be updated to zc core if the sender_ids are not the same but we can move it to be inside the if block.

Reply:
@mykie88 Yeah I understood what you're asking. 

It didn't seem particularly relevant since all it's doing is specifying the payload data, but if it's an issue for you, it can be moved. It doesn't affect the outcome either way

Reply:
oh yeah @man-of-mind it is not a major issue except for the little extra memory allocation wasted during the swap if the check eventually fails downstream.
Pardon me if I appear to be too pedantic. The code can be left as is then.
well done @pauline-banye

Reply:
Please kindly mark conversation as resolved @pauline-banye

Reply:
Yeah, you are right boss @mykie88 , i suggest we move it inside the if block.

Reply:
Like @mykie88, I also think that the swap should come after the if check. This may seem particularly irrelevant, but from a logical point of view, it makes sense.
@pauline-banye

Reply:
Adjusted @mykie88